pgsql/pointcloud.sql.in

```diff
@@ -198,6 +198,12 @@ CREATE OR REPLACE FUNCTION PC_EnvelopeAsBinary(p pcpatch)
 	RETURNS bytea AS 'MODULE_PATHNAME', 'pcpatch_envelope_as_bytea'
 	LANGUAGE 'c' IMMUTABLE STRICT;
 
+-- PC_Envelope is an alias to PC_EnvelopeAsBinary. It is kept for
+-- backward-compatibility reasons.
+CREATE OR REPLACE FUNCTION PC_Envelope(p pcpatch)
+	RETURNS bytea AS 'MODULE_PATHNAME', 'pcpatch_envelope_as_bytea'
+	LANGUAGE 'c' IMMUTABLE STRICT;
+
```

Comment:
I don'tknow how to properly do it. I tried to use PL/pgSQL's [`RAISE`](https://www.postgresql.org/docs/9.6/static/plpgsql-errors-and-messages.html#PLPGSQL-STATEMENTS-RAISE) statement. But it would obviously output the deprecation message on each record passing record passing through the function.